Library/Homebrew/test/api/internal_tap_json/formula_spec.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+describe "Internal Tap JSON -- Formula" do
+  let(:internal_tap_json) { File.read(TEST_FIXTURE_DIR/"internal_tap_json/homebrew-core.json").chomp }
```

Comment:
Would prefer to have this specified inline. I know it's relatively large but it'll help us review the actual generated contents more than a single line fixture.

Reply:
You can just go look at the file though, right?

Reply:
@apainintheneck Technically, yeh, but then I need to mentally convert a JSON string to a Ruby hash. If it was at least pretty-printed in that file (indented, line breaks), it'd make it less painful to e.g. drop comments on that file where stuff shouldn't be present.

Reply:
(some of the constants change between tests/test runners and others stay the same. some tests likely are modifying shared state when they shouldn't)